osu.Game/Rulesets/Mods/ModEasy.cs

```diff
@@ -26,5 +28,21 @@ public void ApplyToDifficulty(BeatmapDifficulty difficulty)
             difficulty.DrainRate *= ratio;
             difficulty.OverallDifficulty *= ratio;
         }
+
+        public void ApplyToScoreProcessor(ScoreProcessor scoreProcessor)
+        {
+            Lives = 2;
+            scoreProcessor.Health.ValueChanged += valueChanged =>
+            {
+                if (scoreProcessor.Health.Value == 0)
```

Comment:
You can combine the two conditions into one.
As in: `scoreProcessor.Health.Value == scoreProcessor.Health.MinValue && lives > 0`